packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -353,6 +353,7 @@ plan-details-show-button = Show details
 plan-details-hide-button = Hide details
 plan-details-total-label = Total
 plan-details-list-price = List Price
+plan-details-tax = Taxe = Taxes & fees
```

Comment:
Is `Taxe = ` wanted here or a mistake?

Reply:
ah mistake, will fix now

Reply:
fixed

Reply:
Sorry, another thought is that you might want `Taxes and fees`, at least based on the content team's guidelines for Firefox: https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Punctuation#Punctuation-Ampersand(&)

Reply:
i was working off of some figma mockups, i will update this to not use the ampersand

Reply:
removed